docs/wiki/featurex/fortify.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: wiki
+title: fortify
+description: enables player to build fortifications.
+group: feature
+category: interaction
+parent: wiki
+mod: acex
+version:
+  major: 3
+  minor: 3
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 Fortification
+
+Enable players to place down fortifications.
+
+### 1.2 Budget
+
+Enables mission makers / admin to restrict the amount of resources available.
+
+## 2. Usage
+Please note that to use this function the `fortify module` needs to be placed down and `fortify tool` equipped by player.
+
+### 2.1 Placing fortification
+
+- Open self-interaction menu with <kbd>Ctrl</kbd> + <kbd>⊞ Win</kbd> (ACE3 default).
+- Select equipment → fortify.
+- Choose your fortification object and follow the instructions on the screen.
+
+### 2.1 Removing fortification
+
+- Use interaction <kbd>⊞ Win</kbd> on object.
+
+## 3. Tips
+
+- Using <kbd>alt</kbd> / <kbd>ctrl</kbd> / <kbd>shift</kbd> + <kbd>scroll wheel</kbd> you can rotate object on different axis.
+- You can use planks to get up on rooftops.
+- Holding <kbd>ctrl</kbd> while placing allows for placing multiple objects of the same type.
+
+## 4. Dependencies
+
+{% include dependenciesx_list.md component="fortify" %}

```

Comment:
You will also want to update `depdenciesx_list.md`, since the one from ACEX is not automatically updated. When Fortify will be merged into ACEX, clone ACEX and run the following from ACE3 `tools`:
```
python3 extract_dependencies.py --markdown --acex <path_to_acex>
```